cawemo-to-web-modeler-migration/migrate-cawemo-to-web-modeler.sh

```diff
@@ -0,0 +1,212 @@
+#!/bin/sh 
+
+set -eu -o pipefail
+
+# ℹ️ Requires Bash 4 or newer, or zsh
+# Recommended to run it on a test environment first
+
+# See these docs to obtain your Cawemo API credentials: https://docs.camunda.org/cawemo/1.9/reference/rest-api/overview/authentication/
+CAWEMO_USER_ID=ENTER_HERE
+CAWEMO_API_KEY=ENTER_HERE
+
+# See these docs to obtain your Web Modeler client credentials: https://docs.camunda.io/docs/next/apis-tools/web-modeler-api/#authentication
+MODELER_CLIENT_ID=ENTER_HERE
+MODELER_CLIENT_SECRET=ENTER_HERE
+
+# Color definitions for terminal output formatting
+GREEN='\033[0;32m'
+NC='\033[0m' # No Color
+B=$(tput bold)
+N=$(tput sgr0)
+BGREEN=$GREEN$B
+NNC=$NC$N
+
+echo "------------------------------------------------------------------------"
+echo "${BGREEN}MIGRATION FROM CAWEMO TO WEB MODELER${NNC}"
+echo "Files are now migrated file-by-file..."
+echo "------------------------------------------------------------------------"
+
+echo ""
+
+echo "[$(date +'%Y-%m-%d %H:%M:%S')] Migration started" >> migration.log
+
+HTTP_STATUS=$(curl -s -o .curl_tmp -w "%{http_code}" -u $CAWEMO_USER_ID:$CAWEMO_API_KEY "https://cawemo.com/api/v1/projects")
```

Comment:
@christian-konrad 
For me, the script logs:
```log
[2023-07-31 08:00:00] Migration started
[2023-07-31 08:00:01] ⚠ GET https://cawemo.com/api/v1/projects failed with status code 404
```

The fetched body looks like this:
```json
{"title":"Not Found","status":404,"detail":"Could not find the requested resource."}
```

Reply:
Resolves as soon as you admin of the org...